Packs/McAfee-TIE/Integrations/McAfee-TIE/CHANGELOG.md

```diff
@@ -1,13 +1,5 @@
 ## [Unreleased]
--
-
-
-## [20.4.0] - 2020-04-14
```

Comment:
don't remove those sections. they are good indications for changes in the integration files